### ISSUE TITLE
Gitlab links to issues in code are busted when running sast analysis via pipeline

### ISSUE THREAD

**Issue by juegge** [checkmarx-ltd/cx-flow#678]
### Description

When a  sast analysis is run using a pipeline in Gitlab, the the links to the code locations of the respective issues are busted. They do not point to the proper location. 

This does not happen if CxFlow is triggered via Webhook and CxFlos is run on a diferent machine (not a container). Other than that both CxFlow versions are 1.6.19

### Expected Behavior
Link points to proper location
Example: https://gitlab.com/jurgen-cx/gitlab-cxflow/cxflow_gitlab_webhook_lab/blob/master/root/password.jsp#L10
Project: https://gitlab.com/jurgen-cx/gitlab-cxflow/cxflow_gitlab_webhook_lab 

### Actual Behavior
Link is busted:
Example: https://gitlab.com/jurgen-cx/gitlab-cxflow/gitlabpipelinetest/-/blob/master/null#L10

### Reproduction

Basically follow the steps outlined in the CxFlow Gitlab lab: https://github.com/checkmarx-ltd/cx-flow/wiki/Labs#cliprep
The issue occurs for  pipeline scans but not for webhook triggered scans

Detail the steps taken to reproduce the issue

Pipeline project: https://gitlab.com/jurgen-cx/gitlab-cxflow/gitlabpipelinetest
This works as is

Webhook project: https://gitlab.com/jurgen-cx/gitlab-cxflow/cxflow_gitlab_webhook_lab
This requires setting up webhooks and running CxFlow on a separate machine

**Comment by jbrotsos** [comment]
Hey @MithileshPawar , this is a regression that I brought up earlier.  Did you see what needs to be done?

**Comment by MithileshPawar** [comment]
> Hey @MithileshPawar , this is a regression that I brought up earlier. Did you see what needs to be done?

I will check what's causing this problem.

**Comment by juegge** [comment]
Any news on this?

It also happens in Github same issue

**Comment by juegge** [comment]
apparently fixed at least since 1.6.22
